phpstan.neon.dist

```diff
@@ -0,0 +1,31 @@
+parameters:
+    level: max
+    checkMissingIterableValueType: false
+    treatPhpDocTypesAsCertain: false
+    paths:
+        - src
+    ignoreErrors:
+        -
+            message: "#^Strict comparison using \\!\\=\\= between null and null will always evaluate to false\\.$#"
+            count: 1
+            path: src/HttpMethodsClient.php
+
+        -
+            message: "#^Cannot call method createStream\\(\\) on Psr\\\\Http\\\\Message\\\\StreamFactoryInterface\\|null\\.$#"
+            count: 1
+            path: src/HttpMethodsClient.php
+
+        -
+            message: "#^Method Http\\\\Client\\\\Common\\\\Plugin\\\\Journal\\:\\:addSuccess\\(\\) has no return typehint specified\\.$#"
+            count: 1
+            path: src/Plugin/Journal.php
+
+        -
+            message: "#^Method Http\\\\Client\\\\Common\\\\Plugin\\\\Journal\\:\\:addFailure\\(\\) has no return typehint specified\\.$#"
```

Comment:
This is a no-fix. Changing it would break BC. Maybe we should consider adding void return to the interface in the next major version.